### PR TITLE
New package: qtikz

### PR DESCRIPTION
**Pull request by Piraty**
This is a somewhat useful tool for rapid prototyping diagrams of all kinds, using the tikZ/LaTeX method.

arm compilation breaks with (https://travis-ci.org/Piraty/void-packages/jobs/256181568#L1044):
```
app/tikzeditor.cpp:322:58: error: no matching function for call to 'qMax(qreal, double)'

  tabulatorsPen.setWidthF(qMax(qreal(0.5), spaceWidth * .1));
```
So i marked it broken for now.

### CODE REVIEW

File: srcpkgs/qtikz/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+makedepends="poppler-qt4-devel qt-devel"
+short_desc="Editor for graphics set in TikZ"
+maintainer="Piraty <piraty@users.noreply.github.com>"
+license="GPL-2"
+homepage="http://www.hackenberger.at/blog/ktikz-editor-for-the-tikz-language/"
+distfiles="https://github.com/fhackenberger/ktikz/archive/${_githash}.zip"
+checksum=6172539872b47f14ccdd5fa5ad0592edfd1d84d25af8769b29bd38429558d420
+
+case "$XBPS_TARGET_MACHINE" in
+	arm*) broken="QT cannot be cross-compiled, sorry.";;
```
Comment (ebfe) [inline srcpkgs/qtikz/template]: This isn't true anymore.

**Comment by ebfe** [comment]
> tabulatorsPen.setWidthF(qMax(qreal(0.5), spaceWidth * .1));

Both args need to have the same type.

File: srcpkgs/qtikz/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+version=20170404
+revision=1
+_githash="be66c8b1ff7e6b791b65af65e83c4926f307cf5a"
+wrksrc=ktikz-${_githash}
+#only_for_archs="i686 x86_64"
+build_style=qmake
+hostmakedepends="unzip qt-qmake pkg-config qt-plugin-sqlite qt-devel"
+makedepends="poppler-qt4-devel qt-devel"
+short_desc="Editor for graphics set in TikZ"
+maintainer="Piraty <piraty@users.noreply.github.com>"
```
Comment (ebfe) [inline srcpkgs/qtikz/template]: srcpkgs/qtikz/template:12: maintainer needs a valid address for sending mail

**Comment by Piraty** [comment]
@ebfe done.

**Comment by ebfe** [comment]
Anything wrong with the tarball at http://www.hackenberger.at/ktikz/ktikz_0.10.tar.gz?
If yes, prefix the version with "0.0." so we can switch to released version without a "replaces=" later.

**Comment by Piraty** [comment]
Yes, tarball's from 2010.
prefix: done @ebfe

**Comment by Duncaen** [comment]
```
=> ERROR: SHA256 mismatch for 'be66c8b1ff7e6b791b65af65e83c4926f307cf5a.tar.gz:'
48715746c91e7010bb4eaf23e1fd668a7b7916155d66028da9d13a9d63468bcf
=> ERROR: qtikz-0.0.20161122_1: couldn't verify distfiles, exiting...
```

**Pull request closed without merging**